sdk/src/RealUserMonitor.ts

```diff
@@ -195,15 +203,22 @@ export default class RealUserMonitor {
    * @param timeAt - The time at which this event occurred, defaults to NOW if undefined/null.
    */
   async transmitRealUserMonitoringEvent(eventName: string, data: Record<string, any>, timeAt?: number) {
+    //Check whether the session has been idle long enough to rotate it
+    if (Date.now() - this.lastSessionInteractionTime > SessionRotateThreshold) await this.rotateRUMSession();
+    else this.markSessionInteraction();
+
     const rumMessage = this.generateRealUserMonitorPayload(eventName, data, timeAt);
 
-    return fetch(this.customRealUserMonitoringEndpoint || this.RAYGUN_RUM_ENDPOINT + '?apiKey=' + encodeURIComponent(this.apiKey), {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify({ eventData: [rumMessage] })
-    }).catch(err => {
+    return fetch(
+      this.customRealUserMonitoringEndpoint || this.RAYGUN_RUM_ENDPOINT + '?apiKey=' + encodeURIComponent(this.apiKey),
```

Comment:
Just use "RUMEndpoint" variable and whenever it is set to null or empty, just overwrite with the RAYGUN_RUM_ENDPOINT

Reply:
Could use a template literal here as well, rather than concatenation

```ts
`${this.customRealUserMonitoringEndpoint}?apiKey=${encodeURIComponent(this.apiKey)`
```